api/db-migrations/000011_add_pyfunc_config_env.down.sql

```diff
@@ -0,0 +1,9 @@
+-- Migrate pyfunc_config data from new schema to old schema (remove the env column)
```

Comment:
Tip: this can be simplified using the `-` operator like: https://stackoverflow.com/a/35283553